v22.2/configure-logs.md

```diff
@@ -112,6 +113,7 @@ All supported sink types use the following common sink parameters:
 | `redactable`    | When `true`, preserves redaction markers around fields that are considered sensitive in the log messages. The markers are recognized by [`cockroach debug zip`](cockroach-debug-zip.html) and [`cockroach debug merge-logs`](cockroach-debug-merge-logs.html) but may not be compatible with external log collectors. For details on how the markers appear in each format, see [Log formats](log-formats.html).             |
 | `exit-on-error` | When `true`, stops the Cockroach node if an error is encountered while writing to the sink. We recommend enabling this option on file sinks in order to avoid losing any log entries. When set to `false`, this can be used to mark certain sinks (such as `stderr`) as non-critical.                                                                                                                                        |
 | `auditable`     | If `true`, enables `exit-on-error` on the sink. Also disables `buffered-writes` if the sink is under `file-groups`. This guarantees [non-repudiability](https://en.wikipedia.org/wiki/Non-repudiation) for any logs in the sink, but can incur a performance overhead and higher disk IOPS consumption. This setting is typically enabled for [security-related logs](logging-use-cases.html#security-and-audit-monitoring). |
+| `buffering`     | **New in v22.2** Configures buffering of log messages for the sink, with the following sub-parameters:<br><br><ul><li>`max-staleness`: The maximum time a log message will wait in the buffer before a flush is triggered. Default: `5s`</li><li>`flush-trigger-size`: The number of bytes that will trigger the buffer to flush. Default: `1MiB`</li><li>`max-buffer-size`: The maximum size of the buffer: new log messages received when the buffer is full cause the older messages to be dropped. Default: `50MiB`</li><li>`max-in-flight`: The maximum number of concurrent active buffered flushes before messages start being dropped. Default: `??`</li></ul>`buffering` is enabled by default for [Fluentd-compatible](#output-to-fluentd-compatible-network-collectors) and [HTTP](#output-to-http-network-collectors) log sinks, and disabled by default for [log files](#output-to-files) and the [standard error stream (`stderr`)](#output-to-stderr) log sinks.<br><br>To explicitly disable log buffering, specify `buffering: NONE` instead. This setting is typically disabled for [security-related logs](logging-use-cases.html#security-and-audit-monitoring). See [Log buffering](#log-buffering) for more details and usage.|
```

Comment:
> disabled by default for [log files](#output-to-files)

As discussed in the Q&A, I believe buffering as a concept is enabled by default for file sinks, but is instead covered under the `buffered-writes` config option.